Somehow I can't pick the d2 type for a file

Great work!

Unfortunately, for me, it simply isn't there. Could this be a regression? The `.d2` files also aren't picked up. I don't know what else to do but to install the extension.

I believe this is an issue with the [languages/d2/injections.scm](https://github.com/gabeidx/zed-d2/blob/main/languages/d2/injections.scm)

It appears that Zed does not appreciate `language` and `injection.language` being defined in this way.

```log
2025-04-10T12:48:24.099967-07:00 [ERROR] failed to load language D2:
Error loading injection query

Caused by:
    both language and injection.language captures are present
```

--- a/languages/d2/highlights.scm
+++ b/languages/d2/highlights.scm
@@ -59,3 +59,22 @@
 ;-------------------------------------------------------------------------------
 
 (ERROR) @error
+
+;-------------------------------------------------------------------------------
+; Language-specific aliases for nvim-treesitter
+;-------------------------------------------------------------------------------
+
+(text_block . (raw_text) @markdown)
+
+(text_block
+  (language) @_language
+  (raw_text) @markdown
+  (#eq? @_language "md"))
+
+(text_block
+  (language) @_language
+  (raw_text) @javascript
+  (#eq? @_language "js"))
+
+(line_comment) @comment
+(block_comment) @comment

--- a/languages/d2/injections.scm
+++ b/languages/d2/injections.scm
@@ -1,48 +1,23 @@
-; for tree-sitter
-
+;; Language injection for markdown by default
 (
   (text_block . (raw_text) @injection.content)
   (#set! injection.language "markdown")
 )
 
-(text_block
-  (language) @injection.language
-  (raw_text) @injection.content
+;; Language-specific injection via explicit language node
+(
+  (text_block
+    (language) @injection.language
+    (raw_text) @injection.content)
 )
 
+;; Comment blocks as injectable content
 (
   (line_comment) @injection.content
   (#set! @injection.language "comment")
 )
+
 (
   (block_comment) @injection.content
   (#set! @injection.language "comment")
 )
-
-;; -------------------------------------
-;; overwrite for nvim-treesitter
-; use markdown as default
-(text_block . (raw_text) @markdown)
-
-; add alias for markdown
-(text_block
-  (language) @_language
-  (raw_text) @markdown
-  (#eq? @_language "md")
-)
-
-; add alias for javascript
-(text_block
-  (language) @_language
-  (raw_text) @javascript
-  (#eq? @_language "js")
-)
-
-(text_block
-  (language) @language
-  (raw_text) @content
-)
-
-(line_comment) @comment
-(block_comment) @comment
-

